server/src/hwApi.ts

```diff
@@ -321,4 +321,38 @@ removed from your team and need to be re-added if you wish to participate in fut
       message: 'Github id updated',
     }
   }
+  async cleanupTeam(teamRole: string) {
```

Comment:
If you look  at the other functions in this API, they are all like:

async addUserToTeam(user: UserT, team: TeamT, silent = false) {
async removeUserFromTeam(user: UserT) {

etc. We don't use identifiers here, so instead have it accept `(team: TeamT)`